backend/tests/test_model_recommendations.py

```
@@ -0,0 +1,70 @@
+    def test_recommendations_include_system_profile(self):
+        with patch('backend.model_manager.get_system_profile') as mock_profile:
+            mock_profile.return_value = {
+                'ram_gb_total': 16.0,
+                'ram_gb_available': 12.0,
+                'cpu_cores_logical': 8,
+                'cpu_cores_physical': 4,
+                'disk_gb_free': 50.0,
+            }
+            data = model_manager.get_model_recommendations(max_results=3)
+
+        self.assertIn('system', data)
```

Ruff (0.15.2)

[warning] Use a regular `assert` instead of unittest-style `assertIn`

Replace `assertIn(...)` with `assert ...`

(PT009)